app/assets/stylesheets/application.scss

```diff
@@ -156,7 +156,6 @@ $govuk-breakpoints: (
     }
   }
 }
-
```

Comment:
may not be a big deal, but take changes that do nothing, they add noise to the history and may confuse us if we try to search it.